### PR TITLE
JBEAP-911-part2: Updated cmt files to match the files in the wildfly …

### PR DESCRIPTION
**Pull request by sgilda**
…quickstart version to fix transaction processing

### CODE REVIEW

**Comment by sgilda** [comment]
@rafabene : Would you mind reviewing this change?

File: cmt/src/main/resources/META-INF/persistence.xml [hunk 1]
```diff
@@ -21,15 +21,10 @@
         http://xmlns.jcp.org/xml/ns/persistence
         http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
    <persistence-unit name="primary">
-      <!-- If you are running in a production environment, add a managed 
-         data source, this example data source is just for development and testing! -->
-      <!-- The datasource is deployed as WEB-INF/cmt-quickstart-ds.xml, you
-         can find it in the source at src/main/webapp/WEB-INF/cmt-quickstart-ds.xml -->
-      <jta-data-source>java:jboss/datasources/CMTQuickstartDS</jta-data-source>
```
Comment (rafabene) [inline cmt/src/main/resources/META-INF/persistence.xml:28]: Why did you remove the Datasource definition too ?

File: cmt/src/main/webapp/duplicate.xhtml [hunk 2]
```diff
@@ -22,6 +22,7 @@
 
 <ui:composition template="template.xhtml">
 	<ui:define name="content">
+        <Count />
```
Comment (rafabene) [inline cmt/src/main/webapp/duplicate.xhtml:25]: What is this tag for ?

**Comment by rafabene** [comment]
I reviewed these changes but I didn't understand why you did it...

File: cmt/src/main/resources/META-INF/persistence.xml (hunk 1 above)
Comment (sgilda) [inline cmt/src/main/resources/META-INF/persistence.xml:28]: Because it was removed from the WildFly quickstart. I basically applied most of the changes that were in this commit:
https://github.com/wildfly/quickstart/commit/235ce39ffa0e3fc24e43390a3a0b66df2bc33df2

This file is deprecated and the quickstart seems to work without it.

File: cmt/src/main/webapp/duplicate.xhtml (hunk 2 above)
Comment (sgilda) [inline cmt/src/main/webapp/duplicate.xhtml:25]: I have no idea. It was added by Eduardo. I will ask him.

**Comment by sgilda** [comment]
@rafabene : The cmt transaciton processing doesn't work as it is currently checked in. I was able to get it to compile and deploy. When you access the application, you can view the list of customers, but when you attempt to add one, it gives you a duplicate exception. I figured I would apply he same changes made to the WildFly quickstart and it does seem to work now. Eduardo made the changes and it looked like you approved and merged them, so I figured they were necessary: https://github.com/wildfly/quickstart/commit/235ce39ffa0e3fc24e43390a3a0b66df2bc33df2

I'll ask Eduardo about the changes.

**Comment by sgilda** [comment]
I am closing this pull and plan to issue another with only the required updates.
